docs-v2/integrations/all/aircall-basic/connect.mdx

```diff
@@ -43,6 +42,12 @@ To authenticate using your API credentials, navigate to the Aircall authenticati
 Enter your credentials as follows:
 - User Name: Your API ID
 - Password: Your API Token
-Click "Submit" to complete the authentication.
+Click "Connect" to complete the authentication.
 
 Note: Make sure to match the credentials exactly as shown above - the field names in the form may differ from your Aircall dashboard.
+
+## Going further
+
+<Card title="Connect to Aircall-Basic" icon="link" href="/integrations/all/aircall-basic/connect" horizontal>
+  Guide to connect to Aircall-Basic using Connect UI
```

Comment:
Sorry it was a left over from me, Connect UI should be named Nango Connect (or just Connect)